plugins/splitbutton/plugin.js

```diff
@@ -25,6 +25,14 @@
 			defaultButton;
 
 		if ( face ) {
+			// Split Button can be defined with simpler definition via strings, needed by (#2091).
+			if ( typeof face === 'string' ) {
+				face = CKEDITOR.tools.clone( editor.ui.items[ face ] );
+
+				if ( !face.icon ) {
+					face.icon = face.name || face.command;
```

Comment:
What is the use case for this? If there is no icon in the original button it's fine to leave it without an icon.

Reply:
Actually this one can be removed, but next case is required, explanations below.